addons/website/static/src/js/content/menu.js

```diff
@@ -302,6 +302,7 @@ publicWidget.registry.StandardAffixedHeader = BaseAnimatedHeader.extend({
      */
     destroy() {
         this.$el.css('transform', '');
+        this.$el[0].classList.remove("o_transformed_not_affixed");
```

Comment:
- `this.el` please
- Are you sure all your DOM notification should not be unobserved by the editor? Like what is done for `_adaptToHeaderChange` and many other changes in this file